src/CallKit/CXCompat.cs

```diff
@@ -0,0 +1,446 @@
+		public virtual CXHandle CallHandle {
+			get => throw new PlatformNotSupportedException (Constants.UnavailableOnMacOS);
+			set => throw new PlatformNotSupportedException (Constants.UnavailableOnMacOS);
+		}
+		public virtual string? ContactIdentifier {
+			get => throw new PlatformNotSupportedException (Constants.UnavailableOnMacOS);
+			set => throw new PlatformNotSupportedException (Constants.UnavailableOnMacOS);
+		}
+		public virtual bool Video {
+			get => throw new PlatformNotSupportedException (Constants.UnavailableOnMacOS);
+			set => throw new PlatformNotSupportedException (Constants.UnavailableOnMacOS);
+		}
+	} /* class CXStartCallAction */
+
+	[Obsolete (Constants.UnavailableOnMacOS)]
+	[Register ("CXTransaction", SkipRegistration = true)]
+	public class CXTransaction : NSObject, INSCoding, INSCopying, INSSecureCoding {
+		public override IntPtr ClassHandle => throw new PlatformNotSupportedException (Constants.UnavailableOnMacOS);
+		public CXTransaction (NSCoder coder) => throw new PlatformNotSupportedException (Constants.UnavailableOnMacOS);
+		protected CXTransaction (NSObjectFlag t) : base (t) => throw new PlatformNotSupportedException (Constants.UnavailableOnMacOS);
+		protected internal CXTransaction (IntPtr handle) : base (handle) => throw new PlatformNotSupportedException (Constants.UnavailableOnMacOS);
+		public CXTransaction (CXAction[] actions) => throw new PlatformNotSupportedException (Constants.UnavailableOnMacOS);
+		public CXTransaction (CXAction action) => throw new PlatformNotSupportedException (Constants.UnavailableOnMacOS);
+
+		public virtual void AddAction (CXAction action) => throw new PlatformNotSupportedException (Constants.UnavailableOnMacOS);
+		public virtual NSObject Copy (NSZone? zone) => throw new PlatformNotSupportedException (Constants.UnavailableOnMacOS);
+		public virtual void EncodeTo (NSCoder encoder) => throw new PlatformNotSupportedException (Constants.UnavailableOnMacOS);
+		public virtual CXAction[] Actions => throw new PlatformNotSupportedException (Constants.UnavailableOnMacOS);
+		public virtual bool Complete => throw new PlatformNotSupportedException (Constants.UnavailableOnMacOS);
+		public virtual NSUuid Uuid => throw new PlatformNotSupportedException (Constants.UnavailableOnMacOS);
+	} /* class CXTransaction */
+
+	[Obsolete (Constants.UnavailableOnMacOS)]
+	[Native]
+	public enum CXErrorCode : long {
+		Unknown = 0,
+		Unentitled = 1,
+		InvalidArgument = 2,
+		MissingVoIPBackgroundMode = 3,
+	}
+
+	[Obsolete (Constants.UnavailableOnMacOS)]
+	public static class CXErrorCodeExtensions {
+		public static NSString? GetDomain (this CXErrorCode self) => throw new PlatformNotSupportedException (Constants.UnavailableOnThisPlatform);
+	}
+
+	[Obsolete (Constants.UnavailableOnMacOS)]
+	[Native]
+	public enum CXErrorCodeIncomingCallError : long {
+		Unknown = 0,
+		Unentitled = 1,
+		CallUuidAlreadyExists = 2,
+		FilteredByDoNotDisturb = 3,
+		FilteredByBlockList = 4
```

Comment:
Not, missing ,

Reply:
This is a back port so I'll fix it in another PR